doc/asciidoc/overview/configuration.adoc

```diff
@@ -0,0 +1,200 @@
+[#neo4j_configuration_system]
+== Configuration system overview
+
+[abstract]
+--
+This documentation is valid for Neo4j Streams from versions `3.5.13`, prior versions
+works statically with the properties provided inside the `neo4j.conf` file
+--
+
+Since the version `3.5.13` we introduced a new configuration systems based on
```

Comment:
Make a mini header "Location of Configuration Information", and say it both ways.

For versions prior to blah blah, use neo4j.conf.  For versions as of an after blah blah, use streams.conf.  

Make it super simple.

Reply:
done